### ISSUE TITLE
AttributeError: 'module' object has no attribute 'VariableManager'

### ISSUE THREAD

**Issue by dj-wasabi** [metacloud/molecule#966]
# Issue Type

- Bug report

# Molecule and Ansible details

```
ansible --version: ansible 2.4.0.0
molecule --version: molecule, version 2.1.0
```

- Molecule installation method: pip
- Ansible installation method: pip

# Desired Behaviour

Successful run of the `molecule verify` command.

# Actual Behaviour (Bug report only)

When executing the `molecule verify`, the following error message is shown:
```
    ==================================== ERRORS ====================================
    ____________________ ERROR collecting tests/test_default.py ____________________
    tests/test_default.py:6: in <module>
        os.environ['MOLECULE_INVENTORY_FILE']).get_hosts('all')
    ../../../environment/.virtualenv/environment/lib/python2.7/site-packages/testinfra/utils/ansible_runner.py:150: in __init__
        self.variable_manager = ansible.vars.VariableManager()
    E   AttributeError: 'module' object has no attribute 'VariableManager'
```
This wasn't happening with Molecule `2.0.4`. I'm not sure if I should create an issue here or with testinfra, so you won! :-)

**Comment by retr0h** [comment]
This is an issue with Molecule and Ansible 2.4.  Molecule is currently not 2.4 compatible.  See #960.

**Comment by dj-wasabi** [comment]
Yes, you are right. I downgraded Ansible back to `2.3.20` and it works again!
Thank you!

**Comment by retr0h** [comment]
@dj-wasabi I also believe if you upgrade to a newer version of `ansible-lint` with Ansible 2.4 you will be ok.

**Comment by dj-wasabi** [comment]
Ok, thank you. Will check that too!

**Comment by alexppg** [comment]
It doesn't work, I just test it.

**Comment by jgeusebroek** [comment]
Latest ansible-lint also fails

**Comment by anl** [comment]
Upgrading to testinfra 1.7.1 resolved this for me.  See: https://github.com/philpep/testinfra/issues/249

**Comment by retr0h** [comment]
Molecule 2.4 released to pypi which supports Ansible 2.4.
